ServerCore/Pages/Puzzles/Create.cshtml.cs

```diff
@@ -27,17 +28,21 @@ public IActionResult OnGet()
         [BindProperty]
         public Puzzle Puzzle { get; set; }
 
-        public async Task<IActionResult> OnPostAsync()
+        public async Task<IActionResult> OnPostAsync(int? eventid)
```

Comment:
Should we require having an event id? We probably don't want puzzles that aren't in events